boards/components/src/nfc.rs

```diff
@@ -0,0 +1,60 @@
+//! Component for NFC Tag.
+//!
+//! Usage
+//! -----
+//! ```rust
+//! let nfct = components::nfct::NfcComponent::new(board_kernel, &nrf52840::nfct::NFCT).finalize(());
+//! ```
+
+// Author: Mirna Al-Shetairy <mshetairy@google.com>
+// Last modified: 19/10/2020
```

Comment:
Do we really need a last modified date? Git can tell us that and this is already out of date

Reply:
Yes, my apologies 😅  I was using a rather old file for reference and as my template. I will update it in my upcoming commit 👍